v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-delegating.md

  - blockchain
  - atom
---

> Blockchains that use staking, such as the Cosmos Hub, are primarely operated by a set of actors called [validators](/learn/faq/what-is-a-validator). The status of a validator (active or candidate), as well as its weight is established based on its amount of voting power. Voting power is obtained when ATOM are locked up in the process of staking, and must be granted to a validator or validator candidate. This is called delegating. 

Reviewer: ```suggestion
Blockchains that use staking, such as the Cosmos Hub, are primarily operated by a set of actors called [validators](/learn/faq/what-is-a-validator). The status of a validator (active or candidate), as well as its weight, is established based on the amount of voting power. Voting power is obtained when ATOM are locked up in the process of staking and must be granted to a validator or validator candidate. This action is called delegating. 
```

> In practice, delegating is done automatically when ATOM holders start staking. A single staking transaction needs to be sent for the whole proces of locking up ATOM and delegating to validator(s) via a [wallet]. Users will generally only be asked for the amount of ATOM to stake and the validator(s) they want to delegate to. The Wallet will take care of the rest (i.e. generating and sending the transaction).

Reviewer: ```suggestion
In practice, delegating is done automatically when ATOM holders start staking. A single staking transaction is sent for the whole process of locking up ATOM and delegating to one or more validators using a [wallet]. Generally, you are asked only for the amount of ATOM to stake and the validators they want to delegate to. The Wallet takes care of the rest by generating and sending the transaction.
```

Reviewer: we want to add the URL target link for [Wallet] too

> Note that validators never obtain custody of the ATOM delegated to them. There is no risk for validators to "steal" their delegators ATOM. However, there is a risk for delegated ATOM to be slashed should the validator they are delegated to misbehave. See [What are the risks associated with Staking] for more. 

Reviewer: ```suggestion
Note that validators never obtain custody of the ATOM that is delegated to them. This restriction means that there is no risk for a validator to "steal" their delegator's ATOM. However, there is a risk for delegated ATOM to be slashed if the validator they are delegated to misbehaves. See [What are the risks associated with Staking]. 
```
